style.css

```diff
@@ -382,3 +412,300 @@ textarea {
+  }
+
+  .logo {
+    margin-top: 25px;
+    font-size: 20px;
+  }
+
+  .desktop-menu-list-container {
+    margin-top: 22px;
+  }
+
+  .desktop-menu-list {
+    display: block;
+  }
+
+  .menu {
+    margin-right: 60px;
+  }
+
+  .menu img {
+    display: none;
+  }
+
+  .header-bottom {
+    height: 916px;
+  }
+
+  .header-background {
+    width: 95%;
+    height: 800px;
+    background-image: url(./images/background/background2.svg);
+    margin: 0 auto;
+    background-size: 1600px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  .greeting,
+  .presentation,
+  .connect,
+  .media-icons {
+    width: 920px;
+  }
+
+  .greeting {
+    height: 110px;
+    margin: 220px auto 12px;
+    padding: 0;
+  }
+
+  .presentation {
+    font-size: 20px;
+    margin: 0 auto 12px;
+  }
+
+  .connect {
+    margin: 0 auto 12px;
+    padding: 0;
+  }
+
+  .media-icons {
+    margin: 0 auto 12px;
+  }
+
+  .works {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0 auto;
+  }
+
+  .grid-row {
+    width: 1156px;
+    height: 450px;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+    grid-template-areas:
+      "card project project"
+      "card specs specs"
+      "card description description"
+      "card tools tools"
+      "card button button"
+      "card blank blank";
+    justify-items: start;
+    border: 1px solid #a7aeff;
+  }
+
+  .grid-row-reversed {
+    width: 1156px;
+    height: 450px;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+    grid-template-areas:
+      "project project card"
+      "specs specs card"
+      "description description card"
+      "tools tools card"
+      "button button card"
+      "blank blank card";
+    justify-items: start;
+    border: 1px solid #a7aeff;
+  }
+
+  .grid-row:hover {
+    box-shadow: 0 10px 50px 0 #a7aeff;
+    cursor: pointer;
+  }
+
+  .card {
+    grid-area: card;
+    padding: 0 20px;
+  }
+
+  .project {
+    width: 496px;
+    margin-top: 30px;
+    grid-area: project;
+    display: flex;
+    justify-content: start;
+  }
+
+  .project-name {
+    width: auto;
+    font-size: 40px;
+  }
+
+  .project-name,
+  .project-specs,
+  .project-description,
+  .project-tools,
+  .card-button-style {
+    padding-left: 20px;
+  }
+
+  .specs {
+    width: 496px;
+    grid-area: specs;
+    display: flex;
+    justify-content: start;
+  }
+
+  .project-specs {
+    font-size: 18px;
+  }
+
+  .project-description {
+    width: 496px;
+  }
+
+  .description {
+    width: 496px;
+    grid-area: description;
+    display: flex;
+    justify-content: start;
+  }
+
+  .tools {
+    width: 496px;
+    grid-area: tools;
+    display: flex;
+    justify-content: start;
+    font-size: 12px;
+  }
+
+  .card-button {
+    width: 496px;
+    grid-area: button;
+    display: flex;
+    justify-content: start;
+  }
```

Comment:
- [ ] Your project cards are similar to the Figma template, but they are not responsive and on smaller screen sizes they are cut out off the screen. Please make them responsive so they can fit properly, maintaining the dimensions, on every screen size.
 - [ ] There are some areas inside the project card with different background colors. Please make them consistent and use the white background color for all the cards to match the Figma template.
 - [ ] The whole card has a cursor behavior as pointer, but when the user hovers over the button the cursor behavior is normal. This is inverted behavior. The expected behavior is to be pointer when the user hovers over the button and normal on the rest of the card so the user can know that that button is clickable. Kindly change that.
 - Your design:
 - 
<img width="959" alt="image" src="https://user-images.githubusercontent.com/61919617/213571742-b4ce8204-9eee-4649-8364-aea8bb7137d8.png">